src/main/java/frc/robot/Robot.java

```diff
@@ -138,6 +142,8 @@ public void teleopInit() {
     if (m_autonomousCommand != null) {
       m_autonomousCommand.cancel();
     }
+
+    gameRobotContainer.teleopInitReset().schedule();
```

Comment:
We should guard this by a null check, since it won't be there for other robot modes